src/main/java/com/plivo/api/Plivo.java

```diff
@@ -14,6 +16,12 @@ public static synchronized void init(String authId, String authToken) {
         phloInstance = new PhloRestClient(authId, authToken).getClient();
     }
 
+  public static synchronized void init(String authId, String authToken, OkHttpClient.Builder httpClientBuilder) {
+      System.out.println("New build");
+      plivoInstance = new PlivoClient(authId, authToken, httpClientBuilder);
+      phloInstance = new PhloRestClient(authId, authToken, httpClientBuilder).getClient();
```

Comment:
Not required for PhloRestClient

Reply:
@Koushik-Ayila  made the changes. Please review the changes